### ISSUE TITLE
Add .travis.yml to automate deployment of poms

### ISSUE THREAD

**Issue by chahuistle** [qbicsoftware/parent-poms#1]
I am getting tired of deploying every single time I make a change, so this should be automated under the assumption that releases come only from the master branch and snapshots come only from the development branch.

**Comment by Zethson** [comment]
I suggest that snapshots come from the testing branch.
So far the development branch has been the branch on which every kind of version has been at some point, on testing only versions which should be deployable on portal-testing are pushed.

So: 
master - release
testing - snapshots
development - nothing

**Comment by chahuistle** [comment]
Good idea. I was not aware that the automatic deployment from Jenkins was
done globally and that this was part of QBiC's process.
